code/controllers/subsystem/ticker.dm

```diff
@@ -398,10 +398,19 @@ SUBSYSTEM_DEF(ticker)
 		to_chat(world, "<font color='purple'><b>Tip of the round: </b>[html_encode(m)]</font>")
 
 /datum/controller/subsystem/ticker/proc/check_queue()
+	if(!queued_players.len)
+		return
 	var/hpc = CONFIG_GET(number/hard_popcap)
-	if(!queued_players.len || !hpc)
+	if(!hpc)
+		listclearnulls(queued_players)
+		for (var/mob/dead/new_player/NP in queued_players)
+			to_chat(NP, "<span class='userdanger'>The alive players limit has been released!<br><a href='?src=[REF(NP)];late_join=override'>\>\>Join Game\<\<</a></span>")
```

Comment:
&lt; please

Reply:
this already does that
```
\<         // insert &lt; (less than)
\>         // insert &gt; (greater than)
```
i hate `&lt;` as its hard to read, 

Anywho I just did it even better way.